frameworks/cassandra/universe/config.json

```diff
@@ -89,7 +89,7 @@
         "placement_constraint": {
           "type": "string",
           "description": "The Marathon-style placement constraint string controlling node placement",
-          "default": ""
+          "default": "hostname:MAX_PER:1"
```

Comment:
Exactly, before CNI we effectively enforced this constraint with the port resources. Now we don't consume them, so we need/want to enforce it with placement.